vktrace/vktrace_layer/vktrace_lib_trim.h

```diff
@@ -90,6 +91,8 @@ enum enum_key_state {
 // return if hotkey triggered;
 bool is_hotkey_trim_triggered();
 
+static const uint32_t INVALID_BINDING_INDEX = std::numeric_limits<uint32_t>::max();
```

Comment:
Why is this defined in vktrace_lib_trim.h? It's not specific to trim functionality. Maybe it should be defined in vktrace_common.h?